### ISSUE TITLE
Recoreder2text tool

### ISSUE THREAD

**Issue by pegroman** [uiuc-hpc/Recorder#23]
when executing the tool the following "error" appears:

**incompatible version: file=2.1.8 != reader=2.3.3**

I couldn't find version 2.1.8 in the repository. Is there any solution?

**Comment by wangvsa** [comment]
Hi @pegroman where did you get the traces from? Are they from our dataset https://library.ucsd.edu/dc/object/bb95276921 ?
If so, you can use Recorder 2.2.1 as mentioned in the [doc](https://recorder.readthedocs.io/en/latest/overview.html).

**Comment by pegroman** [comment]
Yes! I tried to use that dataset. 
I tried to install Recorder 2.2.1 but make failed:

**/usr/bin/ld: .libs/librecorder_la-recorder-utils.o:(.bss+0x730): multiple definition of `__recording'; .libs/librecorder_la-recorder-logger.o:(.bss+0xbc0): first defined here
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:417: librecorder.la] Error 1**

Steps:
1. ./autogen.sh
2. ./configure ./configure --prefix=/home/pegroman/recorder_all/recorder/ CFLAGS=-I/usr/local/hdf5/include/ LDFLAGS=-L/usr/local/hdf5/lib/

Maybe hdf5 and mpi wrong version??
OpenMPI 4.1.5
hdf5 1.14.1-2

**Comment by wangvsa** [comment]
Since HDF5 is already in your system path, I believe you don't need to specify it in CFLAGS and LDFLAGS again.
Could you try `CC=mpicc CXX=mpicxx ./configure --prefix=/home/pegroman/recorder_all/recorder`?
If it still fails, can you show me the the version of your compiler (`mpicc --version`)?

**Comment by pegroman** [comment]
The mpicc version is:
**mpicc --version
gcc (GCC) 13.1.1 20230429**

**Comment by wangvsa** [comment]
Okay, I will try to see if I can reproduce the issue.
In the meantime, if it's possible you could try an older version of gcc. We tested several mpi/gcc versions (older than 10.0) and they worked fine.

**Comment by pegroman** [comment]
As you suggested, I switched to a machine running Rocky Linux 8. I was able to install it with no problem
Install the following packages:
sudo yum install -y hdf5 hdf5-devel hdf5-openmpi hdf5-openmpi-devel hdf5-openmpi-static openmpi openmpi-devel
Some variables for MPI:
export PATH=/usr/lib64/openmpi/bin:$PATH
export LD_LIBRARY_PATH=/usr/lib64/openmpi/lib:$LD_LIBRARY_PATH
Finally, I followed the step by step guide to install Recorder
I didn't get to check the version of gcc. It's probably a bit older than the one I originally tried.
Thanks for your time and patience!!
